stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -95,12 +105,13 @@ public fun MessageListHeader(
     Surface(
         modifier = modifier.fillMaxWidth(),
         elevation = 4.dp,
-        color = ChatTheme.colors.barsBackground,
+        color = color,
+        shape = shape
     ) {
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(horizontal = 8.dp),
```

Comment:
Adding vertical padding allows us to use `Modifier.wrapContent()` at the call site where `MessageListHeader` is used without it looking bad.

This is a stopgap measure for the lack of padding customization which should be dealt with in a different way. Possibly by ditching the use of `Surface` all together and letting users dictate shape, color and height through the use of `Modifier` factory extension functions.

Also wrapping content will help mitigate UI clipping when the users change system font sizing through phone settings.